src/AzureIoTHub.Portal.Infrastructure/DevelopmentConfigHandler.cs

```diff
@@ -82,6 +82,8 @@ internal DevelopmentConfigHandler(IConfiguration config)
 
         public override string DbProvider => this.config.GetValue(DbProviderKey, DbProviders.PostgreSQL);
 
+        public override string CloudProvider => this.config[CloudProviderKey];
```

Comment:
Set by default `Azure`

```suggestion
        public override string CloudProvider => this.config.GetValue(CloudProviderKey, "Azure");
```

Reply:
@kbeaugrand It was decided to not set a default value for the CloudProvider but to generate an exception if the configuration was not present (task #1928). 
We still set Azure as default value ?